src/create_df.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
```

Comment:
I don't think this should be it's own file, I would create section in your `patrick_test.ipynb` that does this, so that you can add more when we get more data.

But it doesn't need to be another file in the repo.